transforms/universal/web2parquet/README.md

```diff
@@ -0,0 +1,37 @@
+# Web Crawler to Parquet
+
+This tranforms crawls the web and download files in real-time.
```

Comment:
transform -> transforms 
download -> downloads

Reply:
Done

Reply:
thank you!